app/assets/javascripts/cqm/CQLStatementDependency.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+
+const StatementReferenceSchema = new mongoose.Schema(
+  {
+    library_name: String,
+    statement_name: String,
+  }
+);
+
+const StatementDependencySchema = new mongoose.Schema(
+  {
+    statement_name: String,
+    dependencies: [StatementReferenceSchema],
```

Comment:
Possibly this should still be a hash instead of an array, its looks like the frontend looks up dependencies per statement.